libs/s25main/ai/aijh/AIConstruction.cpp

```diff
@@ -212,7 +212,7 @@ std::vector<const noFlag*> AIConstruction::FindFlags(const MapPoint pt, unsigned
     // When the radius is at least half the size of the map then we may have duplicates that need to be removed
     if(radius >= std::min(aii.gwb.GetSize().x, aii.gwb.GetSize().y))
     {
-        helpers::makeUnique(flags);
+        helpers::makeUniqueStable(flags);
```

Comment:
why not use https://en.cppreference.com/w/cpp/algorithm/unique ?

Reply:
The entries are sorted (by radius which is usually equal to the distance from `pt`) and `std::unique` requires a sorted container (see the new `makeUnique` which does the sort-unique-erase thing)

Nope we are using C++11 with some C++14 as someone wanted to keep the MSVC 2015 support.